src/rard/static/js/project.js

```diff
@@ -405,10 +412,18 @@ function initRichTextEditor($item) {
         }
       },
       renderItem(item, searchTerm) {
-        // allows you to control how the item is displayed
+        // allows you to control how the item is displayed in the list
         let list_display = item.list_display || item.value;
         return `${list_display}`;
       },
+      onSelect(item, insertItem) {
+        if (item.citation) {
```

Comment:
If you apply the suggestion above, this can be simplified since all items should have a citation

Reply:
I thought this was to distinguish eg antiquarians and bibliographies? Bibs have a citation, I didn't think antiquarians did, do they?

Reply:
With the change to views/mention.py above, the ajax_data always includes a citation for each item - it's just that for everything but Bibliographies it's the same as item.value